### PR TITLE
AWS 2.3 Doc fixes

### PR DESCRIPTION
**Pull request by zkayyali812**
Add necessary doc fixes

### CODE REVIEW

**Comment by tpouyer** [comment]
When we refer to the "stack" version we need to use the `-00` as that will be the version shown to customer.  When we talk about ops-image version we should remove the `-00`

**Review by chrisahl — CHANGES_REQUESTED** [review]
I don't agree with deleting the -00 from the deployment version

File: stories/topics/proc-aws-upgrade-pull-container-image.adoc [hunk 1]
```diff
@@ -7,7 +7,7 @@
 +
 [NOTE]
 ====
-The Ansible on Clouds operational image tag must match the version that you want to upgrade to. For example, if your foundation deployment version is 2.2.20230215, pull the operational image with tag 2.3.20230221 to upgrade to version 2.3.20230221.
+The Ansible on Clouds operational image tag must match the version that you want to upgrade to. For example, if your foundation deployment version is 2.3.20230221, pull the operational image with tag 2.3.20230221 to upgrade to version 2.3.20230221.
```
Comment (leena-jawale) [inline stories/topics/proc-aws-upgrade-pull-container-image.adoc:10]: this should be 
```
The Ansible on Clouds operational image tag must match the version that you want to upgrade to. For example, if your foundation deployment version is 2.2.20230215, pull the operational image with tag 2.3.20230221 to upgrade to version 2.3.20230221
```
The original one. No change required here

**Review by prat98 — APPROVED** [review]
Looks good to me

File: stories/topics/proc-aws-backup-and-restore.adoc [hunk 2]
```diff
@@ -159,7 +159,7 @@ $ docker pull $IMAGE
 +
 [NOTE]
 =====  
-The `ansible-on-clouds-ops` image tag should match the version of your foundation deployment. For example, if your foundation deployment version is 2.2.20230215-00, pull the `ansible-on-clouds-ops` image with tag 2.2.20230215-00.
+The `ansible-on-clouds-ops` image tag should match the version of your foundation deployment. For example, if your foundation deployment version is 2.2.20230215, pull the `ansible-on-clouds-ops` image with tag 2.2.20230215.
```
Comment (chrisahl) [inline stories/topics/proc-aws-backup-and-restore.adoc:162]: from
foundation deployment version is 2.2.20230215
to
foundation deployment version is 2.2.20230215-00

Comment (zkayyali812) [reply stories/topics/proc-aws-backup-and-restore.adoc]: updated

**Comment by tpouyer** [comment]
this looks good to me.

File: stories/topics/proc-aws-backup-and-restore.adoc (hunk 2 above)
Comment (chrisahl) [reply stories/topics/proc-aws-backup-and-restore.adoc]: Actually I'm not sure right now if this is for upgrade section or backup/restore section?  Need to look more

**Review by prat98 — APPROVED** [review]
LGTM!

File: stories/topics/proc-aws-backup-and-restore.adoc (hunk 2 above)
Comment (zkayyali812) [reply stories/topics/proc-aws-backup-and-restore.adoc]: After further investigation, I dont think this page is linked up properly to display as it was replaced with other backup and restore pages which are configured correctly @chrisahl.
This is likely from our 2.2 doc and isnt linked properly to display in our 2.3 UI, but the 2.3 b&r and upgrade pages already have correct content if im notmistaken.

File: stories/topics/proc-aws-backup-and-restore.adoc [hunk 3]
```diff
@@ -40,7 +40,7 @@ $ docker pull $IMAGE
 
 [NOTE]
 =====  
-The `ansible-on-clouds-ops` image tag should match the version of your foundation deployment. For example, if your foundation deployment version is 2.2.20230215-00, pull the `ansible-on-clouds-ops` image with tag 2.2.20230215-00.
+The `ansible-on-clouds-ops` image tag should match the version of your foundation deployment. For example, if your foundation deployment version is 2.2.20230215-00, pull the `ansible-on-clouds-ops` image with tag 2.2.20230215.
```
[the remark this answers is not in the document]
Comment (zkayyali812) [reply stories/topics/proc-aws-backup-and-restore.adoc:43]: Great note, we can safely ignore this file then since it was split up and updated.

**Review by ariordan-redhat — APPROVED** [review]
Local docs build looks OK

**Comment by zkayyali812** [comment]
Hey @ariordan-redhat,
These doc updates are ready to be merged in and updated again.
Thank you

File: stories/topics/proc-aws-backup-and-restore.adoc (hunk 3 above)
[the remark this answers is not in the document]
Comment (chrisahl) [reply stories/topics/proc-aws-backup-and-restore.adoc:43]: hopefully this file can be deleted if not needed.  No need maintaining a file that is not used

**Review by chrisahl — APPROVED** [review]

File: stories/topics/proc-aws-backup-and-restore.adoc (hunk 3 above)
[the remark this answers is not in the document]
Comment (zkayyali812) [reply stories/topics/proc-aws-backup-and-restore.adoc:43]: I think there might be a few other files like this straggling.
Wonder if we can work with doc to find and remove all of them?

[the remark this answers is not in the document]
Comment (ariordan-redhat) [reply stories/topics/proc-aws-backup-and-restore.adoc:43]: I wrote a script to identify unused modules for the on-prem AAP repo. I'll modify it for the clouds docs and create an archive for unused files.

[the remark this answers is not in the document]
Comment (zkayyali812) [reply stories/topics/proc-aws-backup-and-restore.adoc:43]: Thank you @ariordan-redhat !!

**Pull request merged**